packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -474,41 +475,28 @@ export class CXLMarketingNavElement extends LitElement {
     this._rotateMenuItemSearchListenOn();
   }
 
+  /**
+   * Mobile menu toggle.
+   *
+   * @see https://app.clickup.com/t/3phu3bv
+   * @since 2022.10.18
+   */
+  // eslint-disable-next-line class-methods-use-this
+  _toggleMobileMenu() {
+    const bodyElement = document.querySelector('body');
+
+    if (bodyElement) {
+      bodyElement.classList.toggle('cxl-mobile-menu-opened');
+    }
+  }
+
```

Comment:
`cxl-mobile-menu-opened` is a random new token name - when you inspect body classlist, you would have no idea where it came from.

Even more importantly, what could be a solution that is self-contained to cxl-marketing-nav, doesn't pollute global spaces?